app/Console/Commands/UpdateInitiativeBudgetEur.php

```diff
@@ -58,4 +53,23 @@ public function handle()
         $this->info('done!');
     }
 
+    private function getEURExchangeRateFor($projectCurrency, $date): ?ExchangeRate
```

Comment:
As per our verbal discussion, a few questions for this function:
1. It does not return $exchangeRate if we can find it
2. What if the project start day is before 2020-01-01 at the very beginning...?